examples/basic/webpack.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
 // This will make the redux-simpler-router module resolve to the
 // latest src instead of using it from npm. Remove this if running
 // outside of the source.
-var src = path.join(__dirname, '..', '..', 'src')
+var src = path.join(__dirname, '..', '..', 'lib')
```

Comment:
Please remove this change. We want to pull the latest source, not the latest built source. We are already running it through webpack and it'll build it again.